relay-server/src/metrics_extraction/spans/mod.rs

```diff
@@ -295,6 +295,8 @@ fn sanitized_span_description(
         if !module.starts_with("http") {
             return None;
         }
+    } else {
+        return None;
     }
```

Comment:
It would be shorter and better instead of:

```rust
if let Some(module) = module {
        if !module.starts_with("http") {
            return None;
        }
    } else {
        return None;
    }
```

to try to write something like:

```rust
      if !module?.starts_with("http") {
          return None;
      }
```
I guess it should work.

Notice `?` after `module`, which will try to unwrap the module or return `None`.